.ruby-version

```diff
@@ -1 +0,0 @@
-2.7.4
```

Comment:
why was this removed?

Reply:
This was somehow incompatible with AWS CodeBuild and the Ruby 2.7 runtime it uses. CodeBuild didn't like us specifying a patch version (.4). Now that the rest of the Pipeline is working, I'm going to try and get this rbenv configuration integrated back into the project.